docuploader/protos/metadata.proto

```diff
@@ -65,4 +65,7 @@ message Metadata {
     // `{stem}/{version}`.
     string serving_path = 10;
 
+    // The DocFX xref files and xref services required for these docs.
```

Comment:
```suggestion
    // The DocFX xref URLs and xref service URLs required for these docs.
```

(Unless we actually provide a way of supplying xref files, they'll always be URLs.)

Reply:
(It might be worth documenting each field separately, so that in generated code each one gets an appropriate comment. Only a nice-to-have though.)

Reply:
Done with both.